README.md

```diff
@@ -18,11 +18,57 @@ In your `Podfile`, add the following entry:
 pod 'ChartboostMediationAdapterReference'
 ```
 
-## Chartboost Mediation Custom Adapter Implementation Guidelines
+## Chartboost Mediation Custom Adapter Implementation Guide
 
 > [!IMPORTANT]
 > Chartboost Mediation will not be providing official support for any custom adapters. For official adapters that we develop and support, [visit this site](https://adapters.chartboost.com).
 
+1. Create a new class and conform to Helium’s `PartnerAdapter` protocol.
+2. Implement var `partnerSDKVersion: String { get }` with the version of the partner SDK. Most adapters derive this from the partner SDKs API so that it always reports the correct number, even when a newer version is imported.
+3. Implement `var adapterVersion: String { get }` with the version of the mediation adapter. The adapter version format is `<Chartboost Mediation major version>.<Partner major version>.<Partner minor version>.<Partner patch version>.<Partner build version>.<Adapter build version>`.  
+`<Partner build version>` is optional, and omitted by most partners."
+For example, if this adapter is compatible with Helium SDK 4.x and partner SDK 1.2.3.x, and this is its initial release, then adapterVersion is 4.1.2.3.x.0.
+4. Implement var `partnerIdentifier: String { get }` with the internal identifier that the Helium SDK can use to refer to the current partner. Must match the value used in Chartboost Mediation's backend.
+5. Implement `var partnerDisplayName: String { get }` the partner name as it should appear in text.
+6. Implement `func setGDPR(applies: Bool?, status: GDPRConsentStatus)`,  
+`func setCCPA(hasGivenConsent: Bool, privacyString: String)`,  
+and `func setCOPPA(isChildDirected: Bool)`,  
+which receive privacy settings from Chartboost Mediation SDK and apply them to the partner SDK.  
+They are always called at startup when Chartboost Mediation initalizes adapters. They will also be called any time the publisher updates privacy settings on the Mediation SDK.  
+These privacy methods aren't called until *after* `setUp()`, so if your SDK requires privacy settings to be passed in at init then you will need to use default values on the very first launch and store the settings received by these methods for use on subsequent startups.
```

Comment:
```suggestion
These privacy methods aren't called until *after* `setUp()`, so if your SDK requires privacy settings to be passed in at init, then you will need to use default values on the very first launch, and store the settings received by these methods for use on subsequent startups.
```

Reply:
Neither of those commas feel right to me. I don't remember enough formal grammar to explain why, but fwiw Grammarly says the original is ok and that the added comma after "launch" is wrong (but it seems fine with or without the one following "init" so... can both ways be right?)

I suspect that something about the way I wrote this line feels off to you and the commas were an attempt to make it clearer. If you can put your finger on what's awkward about the current phrasing I'll try a different way of saying it.

Reply:
Yeah, I'm guessing it's that both ways are technically grammatically correct (minus the second comma in my version). IMO the original version just feels too long to read all at once without being broken up by at least one comma.

Reply:
I broke it up into three sentences in [this new commit](https://github.com/ChartBoost/chartboost-mediation-ios-adapter-reference/pull/50/commits/57477d26ab012578685085a6523730dbc9ccb1f2).